vision/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -124,7 +124,7 @@ public void testLandmarksGcs() throws Exception {
   @Test
   public void testLandmarksUrl() throws Exception {
     // Act
-    String[] args = {"landmarks", "https://pbs.twimg.com/media/C4wfMOhVcAAUAgq.jpg"};
+    String[] args = {"landmarks", "http://wheresgus.com/landmark.jpg"};
```

Comment:
I believe the project that this repo uses in circle is `java-docs-samples-tests`.

Are you referring the errors found at https://circleci.com/gh/GoogleCloudPlatform/java-docs-samples/388?
```
Running com.example.vision.DetectIT
May 12, 2017 11:22:35 PM io.netty.util.internal.MacAddressUtil defaultMachineId
WARNING: Failed to find a usable hardware address from the network interfaces; using random bytes: 59:89:0f:b8:80:fa:c5:65
Tests run: 21, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 35.985 sec <<< FAILURE! - in com.example.vision.DetectIT
testLandmarksUrl(com.example.vision.DetectIT)  Time elapsed: 2.071 sec  <<< FAILURE!
java.lang.AssertionError: 
Not true that <"Error: image-annotator::Malformed request.: The URL does not appear to be accessible by us. Please double check or download the image and pass it in.
"> contains <"Palace of Fine Arts">
	at com.example.vision.DetectIT.testLandmarksUrl(DetectIT.java:134)
Caused by: java.lang.AssertionError: 
Not true that <"Error: image-annotator::Malformed request.: The URL does not appear to be accessible by us. Please double check or download the image and pass it in.
"> contains <"Palace of Fine Arts">
	at com.example.vision.DetectIT.testLandmarksUrl(DetectIT.java:134)
```